### PR TITLE
Storage Pouches

### PR DESCRIPTION
**Pull request by Geevies**
* Added storage pouches, small storage items that can fit in pockets and webbing, that can hold up to seven small items.
* Added medical pouches to the Nanomed Plus vendors, capable of holding some small medical equipment.
* Added circuit pouches to the EngiVend, capable of holding various circuitboards, electronics, etc.

image

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Review by Alberyk — APPROVED** [review]

File: code/game/objects/items/weapons/storage/pouch.dm [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	icon_state = "pouch"
+	contained_sprite = TRUE
+	use_sound = /decl/sound_category/rustle_sound
+	drop_sound = 'sound/items/drop/hat.ogg'
+	pickup_sound = 'sound/items/pickup/hat.ogg'
+	w_class = ITEMSIZE_SMALL
+	max_w_class = ITEMSIZE_SMALL
+	max_storage_space = 14 // 7 small items
+	storage_slots = 7
+	allow_same_size_storage = TRUE
```
Comment (Casper3667) [inline code/game/objects/items/weapons/storage/pouch.dm:14]: So my concern here is that this allows you to store a pouch within another similar pouch. While not a problem at this very moment with how the subtypes are defined, it should probably be proofed for the future whenever someone inevitable uses the base pouches.

**Review by Casper3667 — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weapons/storage/pouch.dm (hunk 1 above)
Comment (NonQueueingMatt) [reply code/game/objects/items/weapons/storage/pouch.dm:14]: agreed

**Comment by NonQueueingMatt** [comment]
Do not agree with pouches being stored in webbings. Additionally, if these are being added, then medkit storage should be nerfed.

**Pull request closed without merging**